python/GangaTest/Framework/utils.py

```diff
@@ -46,7 +46,7 @@ def sleep_until_state(j, timeout=None, state='completed', break_states=None, sle
     current_status = None
     while j.status != state and timeout > 0:
         if not monitoring_component.isEnabled():
-            monitoring_component.runMonitoring( jobs=jobs.select(j.id,j.id), _loadCredentials=False )
+            monitoring_component.runMonitoring( jobs=jobs.select(j.id,j.id))
```

Comment:
I was simply removing an argument from this function so didn't edit the whitespace at all. I'll make a fix to it now though.